virtinst/install/installerinject.py

```diff
@@ -95,8 +100,11 @@ def perform_initrd_injections(initrd, injections, scratchdir):
     """
     Insert files into the root directory of the initial ram disk
     """
-    _perform_generic_injections(injections, scratchdir, initrd,
-            _run_initrd_commands)
+    try:
+        _perform_generic_injections(injections, scratchdir, initrd,
+                _run_initrd_commands)
+    except Exception:  # pragma: no cover
+        raise
```

Comment:
What's the benefit of this try/except block - it seems to be functionally identical to what the existing code bit

Reply:
You're right, it's not needed because any exception raised by `_perform_generic_injections()` will automatically be visible to the caller of `perform_initrd_injections()`.